project/dexter-server/public/AdminSE/adminSECtrl.js

```diff
@@ -0,0 +1,85 @@
+adminSEApp.controller('adminSECtrl', function($scope, $http, $location ,$routeParams, $log) {
+    "use strict";
+
```

Comment:
Common Rule:
Before executing, deleting APIs should be checked whether the user is a proper user(admin) or not.